### PR TITLE
docs(avatar): add WebexAvatar README

### PR DESCRIPTION
**Pull request by lalli-flores**

### CODE REVIEW

File: src/components/WebexAvatar/README.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
...
+## Preview
+To see all the different possible states of the Webex Avatar component, you can run our storybook
+```shell
+  npm start
 ```
+
+## Embed
+1. Create a component adapter from which the data will be retrieved. See [adapters](../adapters). For instance:
+    ```js
+      new jsonAdapter = new PeopleJSONAdapter()
```
Comment (akoushke) [inline src/components/WebexAvatar/README.md]: New does so many things

Comment (akoushke) [reply src/components/WebexAvatar/README.md]: also dont forget to add `;`

File: src/components/WebexAvatar/README.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
 # Webex Avatar Component
+Webex avatar component displays the avatar and status of a _Webex Teams_ user.
 
-Webex avatar component displays the avatar of a _Webex Teams_ user with a given `personID` and `adapter` object.
+![Webex Avatar](./WebexAvatar.png)
```
Comment (akoushke) [inline src/components/WebexAvatar/README.md]: it be nice to center this, but not a blocker.

File: src/components/WebexAvatar/README.md [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+```shell
+  npm start
 ```
+
+## Embed
+1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
+    ```js
+      new jsonAdapter = new PeopleJSONAdapter()
+    ```
+2. Create a component instance by passing the person ID and the component data adapter
```
Comment (akoushke) [inline src/components/WebexAvatar/README.md]: any possibility we can specify the type of the parameters?

File: src/components/WebexAvatar/README.md (hunk 2 above)
Comment (akoushke) [inline src/components/WebexAvatar/README.md]: if we had more than one `png`s, I prefer to have a `imgs/files` directory for each component.

File: src/components/WebexAvatar/README.md [hunk 4]
```diff
@@ -1,9 +1,22 @@
...
-```js
-<WebexAvatar personID="PersonID" adapter={peopleJSONAdapter} />
+## Preview
+To see all the different possible states of the Webex Avatar component, you can run our storybook
+```shell
+  npm start
 ```
+
+## Embed
+1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
```
Comment (adamweeks) [inline src/components/WebexAvatar/README.md]: We should link to the base PeopleAdapter here to show the format.

File: src/components/WebexAvatar/README.md [hunk 5]
```diff
@@ -1,9 +1,22 @@
...
 
-Webex avatar component displays the avatar of a _Webex Teams_ user with a given `personID` and `adapter` object.
+![Webex Avatar](./WebexAvatar.png)
 
-## How to use
-
-```js
-<WebexAvatar personID="PersonID" adapter={peopleJSONAdapter} />
+## Preview
+To see all the different possible states of the Webex Avatar component, you can run our storybook
```
Comment (adamweeks) [inline src/components/WebexAvatar/README.md]: `storybook:`

File: src/components/WebexAvatar/README.md [hunk 6]
```diff
@@ -1,9 +1,22 @@
...
+1. Create a component adapter from which the data will be retrieved (See [adapters](../../adapters)). For instance:
+    ```js
+      new jsonAdapter = new PeopleJSONAdapter()
+    ```
+2. Create a component instance by passing the person ID and the component data adapter
+    ```js
+      <WebexAvatar personID="personID" adapter={jsonAdapter} />
+    ```
+
+The component knows how to manage its data. If anything changes in the data source that the adapter manages, the component will also update on its own.
```
Comment (adamweeks) [inline src/components/WebexAvatar/README.md]: Maybe another link to the PeopleAdapter?

Add newline at the end please

**Comment by akoushke** [comment]
Our intro README file should be updated accordingly as well. The code is outdated. Also I don’t know if you want to add more info about adapters and data on our intro page in this PR or a separate one.

File: src/components/WebexAvatar/README.md (hunk 1 above)
Comment (adamweeks) [reply src/components/WebexAvatar/README.md]:
```suggestion
      const jsonAdapter = new PeopleJSONAdapter();
```

File: src/components/WebexAvatar/README.md (hunk 3 above)
Comment (lalli-flores) [reply src/components/WebexAvatar/README.md]: Added parameter types as part of the instructions

File: src/components/WebexAvatar/README.md (hunk 6 above)
Comment (lalli-flores) [reply src/components/WebexAvatar/README.md]: This was meant to be a general statement to be used throughout all the component READMEs. Like a little reminder

**Comment by lalli-flores** [comment]
> Our intro README file should be updated accordingly as well. The code is outdated. Also I don’t know if you want to add more info about adapters and data on our intro page in this PR or a separate one.

I noticed that too, but I think it should be on a separate PR

File: src/components/WebexAvatar/README.md (hunk 2 above)
Comment (lalli-flores) [reply src/components/WebexAvatar/README.md]: I centered but i don't know if I like it, it's a bit small of an image, so it kinda gets lost

**Pull request merged**